Source/WebKit/UIProcess/Launcher/glib/BubblewrapLauncher.cpp

```diff
@@ -442,6 +442,18 @@ static void bindV4l(Vector<CString>& args)
     }));
 }
 
+static bool enableDeveloperPermissions()
+{
+    static int enabled = -1;
+
+    if (enabled == -1) {
+        const char* env = g_getenv("WEBKIT_ENABLE_UNSAFE_DEVELOPER_PERMISSIONS_IN_SANDBOX");
```

Comment:
I only added it because you like scary names. Anything works for me.

Reply:
The scary name was needed for the environment variable that completely disables the sandbox because users and even applications were using it when they encountered bugs, without understanding the consequences. Understanding that ptrace is probably a sandbox escape(?) and this is probably really dangerous(?), I doubt that's going to be a problem here.